mobile_app_connector/models/request.py

```diff
@@ -58,17 +65,15 @@ def mobile_contact_us(self, parameters, **params):
             'name': question,
             'categ_id': self.env['crm.claim.category'].sudo().search(
                 [('name', '=', source)]).id,
-            'partner_id': contact_id,
-            'user_id': False
+            'partner_id': contact_id or partner.id,
```

Comment:
This will work as well
```suggestion
            'partner_id': partner.id,
```

Reply:
ok!